PokeAlarm/WebhookStructs.py

```diff
@@ -53,9 +54,12 @@ def pokemon(data):
             'lng': float(data['longitude']),
             'move_1_id': int(move_1_id) if move_1_id is not None else 'unkn',
             'move_2_id': int(move_2_id) if move_2_id is not None else 'unkn',
-            'atk':  int(atk) if atk is not None else 'unkn',
+            'atk': int(atk) if atk is not None else 'unkn',
             'def': int(def_) if def_ is not None else 'unkn',
-            'sta': int(sta) if sta is not None else 'unkn'
+            'sta': int(sta) if sta is not None else 'unkn',
+            'height': float(height) if height is not None else 'unkn',
+            'weight': float(weight) if weight is not None else 'unkn',
+            'gender': int(gender) if gender is not None else 'unkn'
```

Comment:
Since we decided to use gender as a single char, can we reduce `unkn` to a single `?`